test-cases/longevity/longevity-aws-custom-d2-workload1-multidc.yaml

```diff
@@ -0,0 +1,175 @@
+test_duration: 255
+n_monitor_nodes: 1
+n_db_nodes: '3 3 3'
```

Comment:
visual nitpick:

since we don't have lots of descriptions for this case
it's better we have the number of nodes (i.e. layout) next to the instance types.

the stress command in the middle,  make you forget what you saw :)

Reply:
Done.